reflex/state.py

```diff
@@ -1032,6 +1093,79 @@ def get_substate(self, path: Sequence[str]) -> BaseState:
             raise ValueError(f"Invalid path: {path}")
         return self.substates[path[0]].get_substate(path[1:])
 
+    def _get_state_manager(self) -> StateManager:
```

Comment:
this can be a static method. Also, should this go inside `app.py` maybe?

Reply:
The sucky thing about putting anything inside app.py is that `app.py` essentially imports the world and causes circular imports.

I'll move it to the top level of `state.py` for now.